packages/matter.js/src/protocol/interaction/InteractionEndpointStructure.ts

```diff
@@ -132,6 +132,30 @@ export class InteractionEndpointStructure {
         return this.resolveGenericElementName(endpointId, clusterId, commandId, this.commands);
     }
 
+    getEndpoint(endpointId: number): Endpoint | undefined {
+        return this.endpoints.get(endpointId);
+    }
+
+    hasEndpoint(endpointId: number): boolean {
+        return this.endpoints.has(endpointId);
+    }
+
+    getClusterServer(endpointId: number, clusterId: number): ClusterServerObj<any, any, any> | undefined {
+        const endpoint = this.endpoints.get(endpointId);
+        if (endpoint === undefined) {
+            return undefined;
+        }
+        return endpoint.getClusterServerById(clusterId);
```

Comment:
NBD but might write as a one-liner: `this.getEndpoint(endpointId)?.getClusterServerById(clusterId)`

Reply:
good idea, change